content/SCALECLIReference/Storage/CLIDisk.md

```diff
@@ -11,7 +11,624 @@ tags:
++-----+-------------------------+-------+-------+--------+-------------+--------+--------+
+| 1   | Raw_Read_Error_Rate     | 100   | 100   | 50     |             | <dict> | <dict> |
+| 5   | Reallocated_Sector_Ct   | 100   | 100   | 1      |             | <dict> | <dict> |
+| 9   | Power_On_Hours          | 100   | 100   | 0      |             | <dict> | <dict> |
+| 12  | Power_Cycle_Count       | 100   | 100   | 1      |             | <dict> | <dict> |
+| 170 | Reserved_Block_Pct      | 100   | 100   | 10     |             | <dict> | <dict> |
+| 171 | Program_Fail_Count      | 100   | 100   | 0      |             | <dict> | <dict> |
+| 172 | Erase_Fail_Count        | 100   | 100   | 1      |             | <dict> | <dict> |
+| 173 | Avg_Block-Erase_Count   | 100   | 100   | 0      |             | <dict> | <dict> |
+| 174 | Unexpect_Power_Loss_Ct  | 100   | 100   | 0      |             | <dict> | <dict> |
+| 183 | SATA_Int_Downshift_Ct   | 100   | 100   | 0      |             | <dict> | <dict> |
+| 184 | End-to-End_Error        | 100   | 100   | 0      |             | <dict> | <dict> |
+| 187 | Reported_Uncorrect      | 100   | 100   | 0      |             | <dict> | <dict> |
+| 188 | Command_Timeout         | 100   | 100   | 0      |             | <dict> | <dict> |
+| 194 | Temperature_Celsius     | 59    | 48    | 0      |             | <dict> | <dict> |
+| 195 | Hardware_ECC_Recovered  | 100   | 100   | 0      |             | <dict> | <dict> |
+| 196 | Reallocated_Event_Count | 100   | 100   | 0      |             | <dict> | <dict> |
+| 197 | Current_Pending_Sector  | 100   | 100   | 0      |             | <dict> | <dict> |
+| 198 | Offline_Uncorrectable   | 100   | 100   | 0      |             | <dict> | <dict> |
+| 199 | UDMA_CRC_Error_Count    | 100   | 100   | 0      |             | <dict> | <dict> |
+| 202 | Percent_Lifetime_Remain | 100   | 100   | 1      |             | <dict> | <dict> |
+| 206 | Write_Error_Rate        | 100   | 100   | 0      |             | <dict> | <dict> |
+| 246 | Total_LBAs_Written      | 100   | 100   | 0      |             | <dict> | <dict> |
+| 247 | Host_Program_Page_Count | 100   | 100   | 0      |             | <dict> | <dict> |
+| 248 | Bckgnd_Program_Page_Cnt | 100   | 100   | 0      |             | <dict> | <dict> |
+| 180 | Unused_Rsvd_Blk_Cnt_Tot | 1     | 1     | 0      |             | <dict> | <dict> |
+| 210 | RAIN_Success_Recovered  | 100   | 100   | 0      |             | <dict> | <dict> |
++-----+-------------------------+-------+-------+--------+-------------+--------+--------+
+```
+{{< /expand >}}
+{{< /expand >}}
+
+### Temperature Command
+
+The `temperature` command returns the temperature of a specified disk.
+
+{{< expand "Checking Disk Temperature">}}
+
+#### Description
+The `temperature` command requires you to specify which disk you want to see the temperature for. You can also set the S.M.A.R.T. `powermode`, and retrieve previous temperatures using the `cache` option.
```

Comment:
What is the required property? is it `name` or some other? Include how to enter the property argument.
```suggestion
The `temperature` command has one required property, `name` to specify which disk you want to see the temperature for. 
`temperature` has two optional properties, `powermode` and `cache`.
Use the optional S.M.A.R.T. property `powermode`  to retrieve ????.
Use `cahce` to retrieve previous temperatures.
```
Not sure what `powermode` does for the user if they enter it. Create a small table for these or just describe in a sentence, and include the syntax for the arguments for each if not the standard text for entering a property argument.